influxdb3_server/src/http.rs

```diff
@@ -2349,7 +2370,24 @@ pub(crate) async fn route_request(
         (Method::GET | Method::POST, all_paths::API_V3_QUERY_INFLUXQL) => {
             http_server.query_influxql(req).await
         }
-        (Method::GET | Method::POST, all_paths::API_V1_QUERY) => http_server.v1_query(req).await,
+        (Method::GET | Method::POST, all_paths::API_V1_QUERY) => {
+            let handler = V1HttpHandler::new(
+                Arc::clone(&http_server.query_executor) as _,
+                Some(http_server.authorizer.upcast()),
+                http_server.common_state.trace_collector(),
+                INFLUXDB3_VERSION.to_string(),
+            )
+            .with_show_databases(ShowDatabases::new(Arc::clone(
+                &http_server.common_state.catalog,
+            )))
+            .with_show_retention_policies(ShowRetentionPolicies::new(Arc::clone(
+                &http_server.common_state.catalog,
+            )));
+            match handler.route_request(req).await {
```

Comment:
i'm surprised to see this potentially return a 200 if there's an error with the statements. 

https://github.com/influxdata/influxdb3_core/blob/c7612d1a8cadb59c0183896524f51ebe44515623/iox_v1_query_api/src/handler.rs#L182-L194